addons/rose/addon/styles/rose/components/form/_input.scss

```diff
@@ -1,10 +1,18 @@
 @use '../../variables/sizing';
 @use '../../utilities/type';
 
+.rose-form  {
```

Comment:
@randallmorey question: to place the [input fields inline](https://www.figma.com/file/LGu4GPYOmN6Sisf9U8YqXi/HCP-Boundary-Beta?node-id=4358%3A54789), I modified the existing style here to accommodate that. let me know if there's a better way to achieve this?

Reply:
Since this is a one-off, I would just override in the admin styles, rather than changing Rose.

Reply:
ty for the feedback. updated!